content/en/aws/elastic-kubernetes-service/index.md

```diff
@@ -359,6 +359,10 @@ $ curl http://eks-service-2.localhost.localstack.cloud:8081/v1
 
 Note that the host names `eks-service-1.localhost.localstack.cloud` and `eks-service-2.localhost.localstack.cloud` both resolve to `127.0.0.1` (localhost), and can hence be used to talk to your service endpoints, and are used inside the Kubernetes load balancer to distinguish between different services.
 
+In a situation, where you are looking to run your Load Balancer on the standard ports such as 80/443, it would not be able to work. Some of these ports may already be occupied on your local machine. For example, by default LocalStack allocates port 443 to expose the APIs via the HTTPS endpoint (`https://localhost.localstack.cloud`).
```

Comment:
```suggestion
In a situation, where you are looking to run your Load Balancer (LB) on the standard ports such as 80/443, it would not be able to work. Some of these ports may already be occupied on your local machine. For example, by default LocalStack allocates port 443 to expose the APIs via the HTTPS endpoint (`https://localhost.localstack.cloud`). Please make sure to expose your LB on a custom, non-standard port.
```